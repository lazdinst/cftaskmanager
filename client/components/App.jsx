import React from 'react';

// Import React Components
import NavigationHeader from './NavigationHeader.jsx';
import MainContainer from './MainContainer.jsx';

// Define the main app class
export const App = () => {
  return (
    <div >
      <NavigationHeader />
      <MainContainer />
    </div>
  );
};

export default App;