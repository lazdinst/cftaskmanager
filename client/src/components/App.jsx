import React from 'react';
import axios from 'axios';

// Import React Components
import NavigationHeader from './NavigationHeader.jsx';
import MainContainer from './MainContainer.jsx';

// Define the main app class
class App extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
    }
  }

  render() {
    return (
      <div>
        <NavigationHeader />
        <MainContainer />
      </div>
    );
  }
}

export default App;