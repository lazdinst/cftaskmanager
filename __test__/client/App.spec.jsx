import React from 'react'
import { shallow, mount } from 'enzyme';
import App from '../../client/components/App.jsx'
import toJson from 'enzyme-to-json';

describe('<App />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<App />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
