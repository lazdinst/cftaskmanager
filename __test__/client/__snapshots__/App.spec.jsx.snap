// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>>APP --- Snapshot +++capturing Snapshot of APP 1`] = `
<div>
  <div>
    <div
      className="ui inverted top fixed menu"
    >
      <div
        className="ui container"
      >
        <a
          className="header item"
          onClick={[Function]}
        >
          CF Task Management App
        </a>
        <a
          className="item"
          onClick={[Function]}
        >
          Talis Lazdins
        </a>
      </div>
    </div>
  </div>
  fuck you
</div>
`;
